Makefile

```
@@ -454,6 +451,12 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 	# Copy everything to vendor directory
 	$(GO) mod vendor
 
+vet: ## Run mattermost go vet specific checks
+	env GO111MODULE=off $(GO) get -u github.com/mattermost/mattermost-govet
+	env GO111MODULE=off $(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license ./...
```

My first review was wrong here. We don't need the environment variable. Sorry
```suggestion
	$(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license ./...
```